CHANGELOG.md

```diff
@@ -1,13 +1,32 @@
 # Change Log - AWS SDK for Android
 
-## [Release TBD](https://github.com/aws/aws-sdk-android/releases/tag/TBD)
+## [Release 2.19.1](https://github.com/aws/aws-sdk-android/releases/tag/release_v2.19.1)
 
-- Model updates
+### New Features
+
+- **AWS Mobile Client**
+  - Added option to specify browser other than the default (Chrome) for HostedUI [#2152](https://github.com/aws-amplify/aws-sdk-android/pull/2152)
+
+### Model updates
   - AWS Simple Token Service [#2132](https://github.com/aws-amplify/aws-sdk-android/pull/2132)
   - AWS Comprehend [#2122](https://github.com/aws-amplify/aws-sdk-android/pull/2122)
   - AWS Transcribe [#2127](https://github.com/aws-amplify/aws-sdk-android/pull/2127)
   - AWS Translate [#2125](https://github.com/aws-amplify/aws-sdk-android/pull/2125)
   - AWS Textract [#2128](https://github.com/aws-amplify/aws-sdk-android/pull/2128)
+  - AWS Rekognition [#2149](https://github.com/aws-amplify/aws-sdk-android/pull/2149), [#2179](https://github.com/aws-amplify/aws-sdk-android/pull/2179)
+  - AWS SNS [#2150](https://github.com/aws-amplify/aws-sdk-android/pull/2150)
```

Comment:
I think some of them go by "Amazon <Service Name>" instead of "AWS <Service Name>". Might wanna double check them before merging.

Reply:
looks like all of the above except for STS go by "Amazon" :)

Reply:
Good catch!

```suggestion
  - AWS Simple Token Service [#2132](https://github.com/aws-amplify/aws-sdk-android/pull/2132)
  - Amazon Comprehend [#2122](https://github.com/aws-amplify/aws-sdk-android/pull/2122)
  - Amazon Transcribe [#2127](https://github.com/aws-amplify/aws-sdk-android/pull/2127)
  - Amazon Translate [#2125](https://github.com/aws-amplify/aws-sdk-android/pull/2125)
  - Amazon Textract [#2128](https://github.com/aws-amplify/aws-sdk-android/pull/2128)
  - Amazon Rekognition [#2149](https://github.com/aws-amplify/aws-sdk-android/pull/2149), [#2179](https://github.com/aws-amplify/aws-sdk-android/pull/2179)
  - Amazon SNS [#2150](https://github.com/aws-amplify/aws-sdk-android/pull/2150)
```